src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -56,12 +64,17 @@ class ServiceTypeAhead extends Component {
   };
 
   render() {
-    const { services } = this.state;
+    const { defaultSelectedItem, services } = this.state;
     // eslint-disable-next-line prettier/prettier
     const renderService = (s) => { return (s && s.name) ? s.name.trim() : ''; };
 
     return (
-      <Downshift onChange={this.handleOnSelect} itemToString={renderService}>
+      <Downshift
+        onChange={this.handleOnSelect}
+        defaultSelectedItem={defaultSelectedItem}
+        itemToString={renderService}
+        key={defaultSelectedItem}
```

Comment:
Downshift sets a state off of its initial props and doesn't re-mount or reinitialize if those props change. The selected item is added to the redux state after the initialization so passing a key forces Downshift to re-mount if an initial state needs to be restored.